charts/refinery/Chart.yaml

```
@@ -2,8 +2,8 @@ apiVersion: v2
 name: refinery
 description: Chart to deploy Honeycomb Refinery
 type: application
-version: 2.8.1
-appVersion: 2.4.1
+version: 2.8.2
+appVersion: 2.4.2
```

Skip 2.4.2 because there's a regression in there; 2.4.3 has the fix.

```suggestion
appVersion: 2.4.3
```